src/workerd/io/compatibility-date.capnp

```diff
@@ -254,4 +254,10 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # WHATWG URL Standard, leading to a number of issues reported by users. Unfortunately,
   # the specCompliantUrl flag did not contemplate the redirect usage. This flag is
   # specifically about the usage in a redirect().
+
+  durableObjectGetExisting @24 :Bool
```

Comment:
This will need to be @25 now. Another Compat flag landed

Reply:
Ah, thanks... just saw the conflict notice at the bottom of the PR and was wondering about that.  Guess I'll try rebasing on `main`.